packages/account/src/Components/api-token/api-token-card.tsx

```diff
@@ -27,7 +20,6 @@ const ApiTokenCard = ({
                         onChange={() => setFieldValue(name, !value)}
                         value={value}
                         className='api-token__checkbox'
```

Comment:
why do we remove defaultChecked?

Reply:
missed out on the`...props` inside `CompositeCheckbox`, removed it because I could not find defaultChecked as a named prop inside `CompositeCheckbox` component. But upon further investigation found out we are including it using ...props and passing to checkbox. Will need to add type for `defaultChecked` as its not mentioned under `TCompositeCheckbox`. Will push the changes and let you know. Thanks